documentation/AZURE.md

```diff
@@ -0,0 +1,66 @@
+Azure credential and setup
+==========================
+
+Setup
+-----
+* Create an application ( [visual guide](https://docs.microsoft.com/en-in/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-active-directory-application) )<br />
+	Search for **App registrations** in the top search bar and select the first result.<br />
+	Select **New application registration**.<br />
+	Fill in the application form:
+
+		Name: StreisandAuth
+		Application type: Web app / API
+		Sign-on URL: http://StreisandAuth
+
+* Ensure that the application can create Azure instances ( [visual guide](https://docs.microsoft.com/en-in/azure/azure-resource-manager/resource-group-create-service-principal-portal#assign-application-to-role) )<br />
+	Search for **Subscriptions** in the top search bar and select the first result.<br />
+	Choose the desired subscription.<br />
+	Select **Access control (IAM)** from the newly opened menu.<br />
+	Add:
+
+		Role: Contributor
+		Select: StreisandAuth
+
+Credentials
+-----------
+File format ( [source](https://docs.ansible.com/ansible/guide_azure.html#storing-in-a-file) ):
+
+		[default]
+		subscription_id=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+		client_id=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+		secret=xxxxxxxxxxxxxxxxx
+		tenant=xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+File location:
+
+		~/.azure/credentials
+
+* Get the **subscription_id**<br />
+	Search for **Subscriptions** in the top search bar and select the first result.<br />
+	The subscription ID will be in the second column of the subscriptions table.
+
+* Get the **client_id** ( [visual guide](https://docs.microsoft.com/en-in/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key) )<br />
+	Search for **App registrations** in the top search bar and select the first result.<br />
+	Select your application from the list: **StreisandAuth**.<br />
+	The client_id is the Guid under **Application ID**.<br />

```

Comment:
Can you define "Guid" quickly? I think this term may have the potential to confuse and isn't introduced anywhere before here.